src/clojure/nrepl/server.clj

```diff
@@ -158,7 +158,7 @@
       (binding [dynamic-loader/*state* state]
         ((:handler @state) msg)))))
 
-(defrecord Server [server-socket port open-transports transport greeting handler]
+(defrecord Server [server-socket host port socket open-transports transport greeting handler]
```

Comment:
I'm wondering if here we shouldn't name this `unix-socket`, to differentiate from the `server-socket`. Not a big deal, but it might make things clearer for someone. 

Or maybe we should add some docstring to this record to explain its purpose.

Reply:
I will:

* Add some explanatory comments to the `defrecord` form re: what the fields are supposed to mean (unfortunately, this won't be a docstring, since `defrecrord` doesn't support them)
* Open an additional PR just like this one, but with the `socket` field renamed to `unix-socket`

And then I'll leave it to you all to choose which one you like better and let you update the comments as you wish :)